packages/ts/src/components/leaflet-map/index.ts

```diff
@@ -386,6 +386,14 @@ export class LeafletMap<Datum extends GenericDataRecord> extends ComponentCore<D
     this.render()
   }
 
+  /** Get the currently expanded cluster */
+  public getExpandedCluster (): { id: string; points: Datum[] } {
+    if (!this._expandedCluster) return
```

Comment:
@reb-dev Since the function can also return `undefined` let's update the return type to `{ id: string; points: Datum[] } | undefined`

Reply:
Fixed!